client/src/Register.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import "./Register.css";
+import search_icon from './assets/search.svg'
+
+export default function Register() {
+    const [nameInput, setNameInput] = React.useState("");
+    const [passwordInput, setPasswordInput] = React.useState("");
+    const [emailInput, setEmailInput] = React.useState("");
+    const [locationInput, setLocationInput] = React.useState("");
+
+    if (nameInput){
+        console.log(nameInput)
+    }
+    if (passwordInput){
+        console.log(passwordInput)
+    }
+    if (emailInput){
+        console.log(emailInput)
+    }
+    if (locationInput){
+        console.log(locationInput)
+    }
+
+    return (
+        <div className = "register">
+            <p className="register_header">Register</p>
+            <div className = "textInputSet">
```

Comment:
nit: might be worth creating a specialized input component since all 4 of these have the exact same format